### DISCUSSION TITLE
Differential gene expression

### DISCUSSION THREAD

**Discussion by rosaluna95** [satijalab/seurat#6576] [General]
Hi, 

I am currently analyzing my single-cell data using Seurat. I am just confused when I analyze differential gene expression using FindMarker function. 

I am getting over 4,000 DEG which is a high number than expected. Does this seem right? I have 18,077 cells and 15,676 features which come to 15,676 features and 2,060 cells when subletting for the findmarker function. 

Thank you,

**Comment by Lea2609** [comment]
4000 DEG with p_val or p_val_adjust ?

**Comment by Jiayi-Zheng** [comment]
Check the p_val_adjusted and the log2fc, if log2fc is too small or p_val_adjusted too large then it's probably not meaningful. Discard it. I don't usually get that many DEGs with all the filtering i added.
